src/pg_timestamp.erl

```diff
@@ -49,9 +49,22 @@ encode_timestamp(infinity) ->
     16#7FFFFFFFFFFFFFFF;
 encode_timestamp('-infinity') ->
     -16#8000000000000000;
+encode_timestamp({{Year, Month, Day}, {Hour, Minute, Seconds}, {HourOffset, MinuteOffset}}) when is_integer(Seconds) ->
+    Sign = determine_sign(HourOffset),
```

Comment:
The offset is either positive or negative. [`calendar:time_to_seconds`](https://www.erlang.org/doc/apps/stdlib/calendar.html#time_to_seconds/1) doesn't work with negative values though.

Reply:
I think this is just wrong?
```
1> calendar:time_to_seconds({0, -100, 0}).
-6000
2> calendar:time_to_seconds({-10, 100, 0}).
-30000
3> calendar:time_to_seconds({-10, 0, 0}).
-36000
4> calendar:time_to_seconds({-100, 0, 0}).
-360000
```